### PR TITLE
add page for partner policies

### PR DESCRIPTION
**Pull request by maneesha**

### CODE REVIEW

File: _extras/partner_policy.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: "Information for Partners"
+permalink: /partners/
```
Comment (jduckles) [inline _extras/partner_policy.md]: Can we change both line 3 and 4's use of "partner" to "member"

File: _extras/partner_policy.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+You must sign up for this training at  least four weeks in advance. Events may be cancelled if no member organizations sign up by four weeks in advance.
+All events run from 9am to 5pm in the time zone listed.  You do not need to be in the same time zone.  However, you do need to accommodate that time zone.  For example, a group from US/Central time may join a training scheduled from 9am to 5pm US/Eastern, but would need to be available from 8am to 4pm their time.
+The number of seats your organization receives depends on your membership level (Silver/Gold/Platinum).  At least six participants must attend at once.  Any unused seats will be forfeited. This is to ensure we can accommodate everyone across all our partner sites. Our  trainings include group exercises that  your  team must work on together.  We also want our training cohorts to recognize they are part of a team which represents the Carpentries at their institution.
+Again, because we have group exercises, the entire team must be in the same room.  Individuals can not participate remotely.
```
Comment (jduckles) [inline _extras/partner_policy.md]: Consider adding **emphasis** to "the entire team must be in the same room"

File: _extras/partner_policy.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+The number of seats your organization receives depends on your membership level (Silver/Gold/Platinum).  At least six participants must attend at once.  Any unused seats will be forfeited. This is to ensure we can accommodate everyone across all our partner sites. Our  trainings include group exercises that  your  team must work on together.  We also want our training cohorts to recognize they are part of a team which represents the Carpentries at their institution.
+Again, because we have group exercises, the entire team must be in the same room.  Individuals can not participate remotely.  
+If an individual misses more than one hour of the training over both days, he/she will not pass the training requirement and his/her spot will be forfeited.  We are training people to deliver Carpentries training and be ambassadors of the Carpentries program, and need to ensure they get our full experience.
```
Comment (jduckles) [inline _extras/partner_policy.md]: "We are training people to deliver Carpentries training and" I'd drop this sentence. It sounds kind of nagging. The point is made above.

File: _extras/partner_policy.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Names and emails of participants must be shared with Carpentries staff at least three weeks before the event.  Participants will be asked to complete our application form to register in our system (https://amy.software-carpentry.org/forms/request_training/).  Participants will receive a link to more information including some pre-reading materials in advance of the event.  
+Exceptions to any of the above may be granted at the discretion of Carpentries staff on a case by case basis. For questions please contact maneesha@carpentries.org.
```
Comment (jduckles) [inline _extras/partner_policy.md]: Consider a role based email address (training@ instructors@ admin@) here.

File: _extras/partner_policy.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+Our trainings are all offered as two day, online events.  You can view our training calendar here (INSERT LINK WHEN WE HAVE ONE) and sign up for a training that meets your team’s needs. A few points as you select dates:
+
+* We recommend that you sign up for an event during the first six months of your partnership year.  This ensures your trainees have time to complete the training program and start teaching workshops during your membership year.
```
Comment (karinlag) [inline _extras/partner_policy.md]: Could we add that we highly recommend that they also try to ensure that the people they want to train have actually attended a workshop before going into training?

File: _extras/partner_policy.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* You must sign up for this training at  least four weeks in advance. Events may be cancelled if no member organizations sign up by four weeks in advance.
```
Comment (karinlag) [inline _extras/partner_policy.md]: ..no member organizations have signed up 4 weeks before the training event.

File: _extras/partner_policy.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+* All events run from 9am to 5pm in the time zone listed.  You do not need to be in the same time zone.  However, you do need to accommodate that time zone.  For example, a group from US/Central time may join a training scheduled from 9am to 5pm US/Eastern, but would need to be available from 8am to 4pm their time.
+* The number of seats your organization receives depends on your membership level (Silver/Gold/Platinum).  At least six participants must attend at once.  Any unused seats will be forfeited. This is to ensure we can accommodate everyone across all our partner sites. Our  trainings include group exercises that  your  team must work on together.  We also want our training cohorts to recognize they are part of a team which represents the Carpentries at their institution.
```
Comment (karinlag) [inline _extras/partner_policy.md]: Suggest rephrasing the number of participants, right now I think it seems a bit confusing. First we say that they need to sign up 6 people, then we say that any unused seats will be forfeited...?

File: _extras/partner_policy.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+* Again, because we have group exercises, the entire team must be in the same room.  Individuals can not participate remotely.  
+* If an individual misses more than one hour of the training over both days, he/she will not pass the training requirement and his/her spot will be forfeited.  We are training people to deliver Carpentries training and be ambassadors of the Carpentries program, and need to ensure they get our full experience.
```
Comment (karinlag) [inline _extras/partner_policy.md]: Does this mean that they can lose 1 hour each day, or 30 minutes each day, or 1 hour total over the 2 days, or...?

File: _extras/partner_policy.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+* Names and emails of participants must be shared with Carpentries staff at least three weeks before the event.  Participants will be asked to complete our application form to register in our system (https://amy.software-carpentry.org/forms/request_training/).  Participants will receive a link to more information including some pre-reading materials in advance of the event.  
+* Exceptions to any of the above may be granted at the discretion of Carpentries staff on a case by case basis. For questions please contact maneesha@carpentries.org.
```
Comment (karinlag) [inline _extras/partner_policy.md]: suggest replacing maneeshas personal email with admin.

File: _extras/partner_policy.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+
+Once you’ve confirmed dates and participants, here is some information about preparing for the event itself:
+
+* Trainings are offered remotely.  Your room should be set up for videoconferencing with a large screen & projector, speakers, camera, and microphone.
```
Comment (karinlag) [inline _extras/partner_policy.md]: also mention breakout rooms for video exercises...?

File: _extras/partner_policy.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+* The training room should have wifi capable of videoconferencing and for each participant laptop to connect.
+* Participants should bring their own laptop - not a Chromebook, tablet, etc.  No special software other than a modern internet browser is required.
+* Schedule will include a lunch break as well as mini breaks during the day.  We strongly encourage each site to provide coffee and lunch to help build team cohesiveness and to maintain the schedule.
+* Carpentries will provide a link to host the videoconference via Blue Jeans in advance of the event in order to test your setup.
+
+
+Lastly, some information about what happens after the event:
+
+* Participants will have three months (90 days) to complete post-training homework exercises required to receive their instructor certificate (http://swcarpentry.github.io/instructor-training/checkout/).   
+* Participants who fail to complete these exercises will not be eligible for instructor certification. Extensions may be granted at the discretion of Carpentries staff.
```
Comment (karinlag) [inline _extras/partner_policy.md]: Extensions may be granted at the discretion of Carpentries staff, but these must be applied for within the 90 days limit.

**Pull request merged**